scripts/get_usernames.py

```diff
@@ -0,0 +1,20 @@
+import csv
+import json
+
+def make_json(csvFilePath, jsonFilePath):
+	data = []
+
+	with open(csvFilePath, encoding='utf=8') as csvf:
+		csvReader = csv.DictReader(csvf)
+
+		for row in csvReader:
+			data.append(row['login'])
+
+	with open(jsonFilePath, 'w', encoding='utf-8') as jsonf:
+		jsonf.write(json.dumps(data, indent=4))
+
+
+csvFilePath = 'four11_map.csv'
```

Comment:
Nit: Constants and string literals should be in all caps
```suggestion
CSV_FILE_PATH = "four11_map.csv"
```